lib/cdo/lesson_import_helper.rb

```diff
@@ -19,34 +19,47 @@ module LessonImportHelper
   # }
   # @param [Lesson] lesson - Code Studio Lesson object to update.
   # @param [Hash] cb_lesson_data - Lesson and activity data to import.
-  def self.update_lesson(lesson, cb_lesson_data = {})
+  def self.update_lesson(lesson, models_to_import = [], cb_lesson_data = {})
     raise unless [:development, :adhoc, :levelbuilder].include? rack_env
     raise unless lesson.script.hidden
 
+    raise "Must specify models to import" if models_to_import.blank?
```

Comment:
this seems like an odd place for this check -- for example, if I do not specify any models, I hit this error. but if I specify `-m LessonGroup`, this check will pass, even though no lesson-related models are being imported. how about moving it up to the main import_unit_details.rb script?